app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/sdk/model/Fido2CredentialRegistrationUserInterfaceImpl.kt

```diff
@@ -15,25 +15,35 @@ import com.x8bit.bitwarden.data.platform.annotation.OmitFromCoverage
 @OmitFromCoverage
 class Fido2CredentialRegistrationUserInterfaceImpl(
     private val isVerificationSupported: Boolean,
-    private val checkUser: suspend (CheckUserOptions, UiHint?) -> CheckUserResult,
-    private val checkUserAndPickCredentialForCreation: suspend (
-        options: CheckUserOptions,
-        newCredential: Fido2CredentialNewView,
-    ) -> CipherViewWrapper,
+    private val selectedCipherView: CipherView,
 ) : Fido2UserInterface {
 
+    /**
+     * Implicitly returns a [CheckUserResult] indicating the user is present and verified.
+     *
+     * The [options] parameter is ignored. User verification is performed prior to invoking the SDK.
+     */
     override suspend fun checkUser(
         options: CheckUserOptions,
         hint: UiHint,
-    ): CheckUserResult = checkUser.invoke(options, hint)
+    ): CheckUserResult = CheckUserResult(true, true)
 
+    /**
+     * Returns a [CipherViewWrapper] containing the [selectedCipherView] the [newCredential] will be
+     * registered to.
+     */
     override suspend fun checkUserAndPickCredentialForCreation(
         options: CheckUserOptions,
         newCredential: Fido2CredentialNewView,
-    ): CipherViewWrapper = checkUserAndPickCredentialForCreation.invoke(options, newCredential)
+    ): CipherViewWrapper = CipherViewWrapper(selectedCipherView)
```

Comment:
We've not updated the SDK in a while. I can take care of that in a separate PR in case it introduces other changes we're not aware of. I'll make a note to track updating it since Renovate can't monitor the artifacts. Appreciate the heads up.